src/BinSkim.Rules/RuleResources.resx

```diff
@@ -151,10 +151,10 @@
     <value>'{0}' is not known to be an obsolete binary that is vulnerable to one or more security problems.</value>
   </data>
   <data name="BA2006_BuildWithSecureTools_Description" xml:space="preserve">
-    <value>Application code should be compiled with the most up-to-date tool sets possible in order to take advantage of the most current compile-time security features.</value>
+    <value>Application code should be compiled with the most up-to-date tool sets possible to take advantage of the most current compile-time security features.</value>
   </data>
   <data name="BA2006_Error" xml:space="preserve">
-    <value>'{0}' was compiled with one or more modules which were not built using minimum required tool versions (compiler version {1}, linker version {2}). More recent tool chains contain mitigations that make it more difficult for an attacker to exploit vulnerabilities in programs they produce. To resolve this issue, compile and/or link your binary with more recent tools. If you are servicing a product where the tool chain cannot be modified (e.g. producing a hotfix for an already shipped version) ignore this warning.
+    <value>'{0}' was compiled with one or more modules which were not built using minimum required tool versions (compiler version {1}, linker version {2}). More recent tools contain mitigations that make it more difficult for an attacker to exploit vulnerabilities in programs they produce. To resolve this issue, compile and/or link your binary with more recent tools. If you are servicing a product where the tool chain cannot be modified (e.g. producing a hotfix for an already shipped version) ignore this warning.
```

Comment:
I changed "tool chain" to "tools". Except in the last sentence, where it just seemed better, but that is the only place in the entire resource file where I did not make that change, and I could be persuaded to change it here too.